### PR TITLE
r/ecs_task_definition - add support for fsx win fs

### LINKED ISSUE #19652: Support ECS FSxWindowsFileServerVolumeConfiguration
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS supports [using FSx volumes for Windows containers](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/wfsx-volumes.html) since [Nov 2020](https://aws.amazon.com/about-aws/whats-new/2020/11/amazon-ecs-supports-use-of-amazon-fsx-windows-file-server/). This is done as a new volume configuration block for task definitions.

- ECS [RegisterTaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_RegisterTaskDefinition.html)
- ECS [Volume](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Volume.html#ECS-Type-Volume-fsxWindowsFileServerVolumeConfiguration)
- ECS [FSxWindowsFileServerVolumeConfiguration](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_FSxWindowsFileServerVolumeConfiguration.html)

### New or Affected Resource(s)

* aws_ecs_task_definition

### Potential Terraform Configuration

```hcl
resource "aws_ecs_task_definition" "service" {
  family                = "service"
  container_definitions = file("task-definitions/service.json")

  volume {
    name = "service-storage"

    fsx_windows_file_server_volume_configuration {
      file_system_id = aws_fsx_windows_file_system.fs.id
      root_directory = "\\data"
      authorization_config {
        credentials_parameter = aws_ssm_parameter.fs_creds.name
        domain                = "ad.example.org"
      }
    }
  }
}
```

### PR DESCRIPTION
**Pull request by DrFaust92**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Closes #19652

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccAWSEcsTaskDefinition_'
--- PASS: TestAccAWSEcsTaskDefinition_inferenceAccelerator (119.06s)
--- PASS: TestAccAWSEcsTaskDefinition_withDockerVolume (120.31s)
--- PASS: TestAccAWSEcsTaskDefinition_withDockerVolumeMinimalConfig (121.41s)
--- PASS: TestAccAWSEcsTaskDefinition_withTaskScopedDockerVolume (122.75s)
--- PASS: TestAccAWSEcsTaskDefinition_arrays (122.84s)
--- PASS: TestAccAWSEcsTaskDefinition_constraint (124.74s)
--- PASS: TestAccAWSEcsTaskDefinition_ExecutionRole (126.04s)
--- PASS: TestAccAWSEcsTaskDefinition_withIPCMode (137.08s)
--- PASS: TestAccAWSEcsTaskDefinition_ProxyConfiguration (138.65s)
--- PASS: TestAccAWSEcsTaskDefinition_withPidMode (139.52s)
--- PASS: TestAccAWSEcsTaskDefinition_withNetworkMode (139.52s)
--- PASS: TestAccAWSEcsTaskDefinition_withTaskRoleArn (139.83s)
--- PASS: TestAccAWSEcsTaskDefinition_Fargate (164.55s)
--- PASS: TestAccAWSEcsTaskDefinition_disappears (165.69s)
--- PASS: TestAccAWSEcsTaskDefinition_withScratchVolume (101.32s)
--- PASS: TestAccAWSEcsTaskDefinition_basic (174.27s)
--- PASS: TestAccAWSEcsTaskDefinition_changeVolumesForcesNewResource (176.17s)
--- PASS: TestAccAWSEcsTaskDefinition_withTransitEncryptionEFSVolume (92.86s)
--- PASS: TestAccAWSEcsTaskDefinition_withEFSVolumeMinimal (95.93s)
--- PASS: TestAccAWSEcsTaskDefinition_withEFSVolume (99.88s)
--- PASS: TestAccAWSEcsTaskDefinition_Tags (230.38s)
--- PASS: TestAccAWSEcsTaskDefinition_withEcsService (251.57s)
--- PASS: TestAccAWSEcsTaskDefinition_withFsxWinFileSystem (3789.91s)
--- PASS: TestAccAWSEcsTaskDefinition_withEFSAccessPoint (51.39s)
```

### CODE REVIEW

File: website/docs/r/ecs_task_definition.html.markdown [hunk 1]
```diff
@@ -222,11 +251,24 @@ For more information, see [Specifying an EFS volume in your Task Definition Deve
...
 * `iam` - (Optional) Whether or not to use the Amazon ECS task IAM role defined in a task definition when mounting the Amazon EFS file system. If enabled, transit encryption must be enabled in the EFSVolumeConfiguration. Valid values: `ENABLED`, `DISABLED`. If this parameter is omitted, the default value of `DISABLED` is used.
 
+### fsx_windows_file_server_volume_configuration
+
+For more information, see [Specifying an FSX Windows File Server volume in your Task Definition Developer Guide](https://docs.amazonaws.cn/en_us/AmazonECS/latest/developerguide/tutorial-wfsx-volumes.html)
+
+* `file_system_id` - (Required) The Amazon FSx for Windows File Server file system ID to use.
+* `root_directory` - (Required) The directory within the Amazon FSx for Windows File Server file system to mount as the root directory inside the host.
+* `authorization_config` - (OptiRequiredonal) Configuration block for [authorization](#authorization_config) for the Amazon FSx for Windows File Server file system detailed below.
```
Comment (ewbankkit) [inline website/docs/r/ecs_task_definition.html.markdown:265]: Optional or required? 😄.

File: website/docs/r/ecs_task_definition.html.markdown [hunk 2]
```diff
@@ -222,11 +251,24 @@ For more information, see [Specifying an EFS volume in your Task Definition Deve
...
 
-### authorization_config
+#### authorization_config
 
 * `access_point_id` - (Optional) Access point ID to use. If an access point is specified, the root directory value will be relative to the directory set for the access point. If specified, transit encryption must be enabled in the EFSVolumeConfiguration.
 * `iam` - (Optional) Whether or not to use the Amazon ECS task IAM role defined in a task definition when mounting the Amazon EFS file system. If enabled, transit encryption must be enabled in the EFSVolumeConfiguration. Valid values: `ENABLED`, `DISABLED`. If this parameter is omitted, the default value of `DISABLED` is used.
 
+### fsx_windows_file_server_volume_configuration
+
+For more information, see [Specifying an FSX Windows File Server volume in your Task Definition Developer Guide](https://docs.amazonaws.cn/en_us/AmazonECS/latest/developerguide/tutorial-wfsx-volumes.html)
```
Comment (ewbankkit) [inline website/docs/r/ecs_task_definition.html.markdown:261]: The link URL looks like the China partition's documentation - Can you change to the Commercial partition (`docs.aws.amazon.com`)?

File: website/docs/r/ecs_task_definition.html.markdown (hunk 1 above)
Comment (DrFaust92) [reply website/docs/r/ecs_task_definition.html.markdown]: fixed

File: website/docs/r/ecs_task_definition.html.markdown (hunk 2 above)
Comment (DrFaust92) [reply website/docs/r/ecs_task_definition.html.markdown]: fixed

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

##### Commercial

```console
% make testacc TEST=./aws TESTARGS='-run=TestAccAWSEcsTaskDefinition_'                  
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSEcsTaskDefinition_ -timeout 180m
=== RUN   TestAccAWSEcsTaskDefinition_basic
=== PAUSE TestAccAWSEcsTaskDefinition_basic
=== RUN   TestAccAWSEcsTaskDefinition_withScratchVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withScratchVolume
=== RUN   TestAccAWSEcsTaskDefinition_withDockerVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withDockerVolume
=== RUN   TestAccAWSEcsTaskDefinition_withDockerVolumeMinimalConfig
=== PAUSE TestAccAWSEcsTaskDefinition_withDockerVolumeMinimalConfig
=== RUN   TestAccAWSEcsTaskDefinition_withEFSVolumeMinimal
=== PAUSE TestAccAWSEcsTaskDefinition_withEFSVolumeMinimal
=== RUN   TestAccAWSEcsTaskDefinition_withEFSVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withEFSVolume
=== RUN   TestAccAWSEcsTaskDefinition_withTransitEncryptionEFSVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withTransitEncryptionEFSVolume
=== RUN   TestAccAWSEcsTaskDefinition_withEFSAccessPoint
=== PAUSE TestAccAWSEcsTaskDefinition_withEFSAccessPoint
=== RUN   TestAccAWSEcsTaskDefinition_withFsxWinFileSystem
=== PAUSE TestAccAWSEcsTaskDefinition_withFsxWinFileSystem
=== RUN   TestAccAWSEcsTaskDefinition_withTaskScopedDockerVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withTaskScopedDockerVolume
=== RUN   TestAccAWSEcsTaskDefinition_withEcsService
=== PAUSE TestAccAWSEcsTaskDefinition_withEcsService
=== RUN   TestAccAWSEcsTaskDefinition_withTaskRoleArn
=== PAUSE TestAccAWSEcsTaskDefinition_withTaskRoleArn
=== RUN   TestAccAWSEcsTaskDefinition_withNetworkMode
=== PAUSE TestAccAWSEcsTaskDefinition_withNetworkMode
=== RUN   TestAccAWSEcsTaskDefinition_withIPCMode
=== PAUSE TestAccAWSEcsTaskDefinition_withIPCMode
=== RUN   TestAccAWSEcsTaskDefinition_withPidMode
=== PAUSE TestAccAWSEcsTaskDefinition_withPidMode
=== RUN   TestAccAWSEcsTaskDefinition_constraint
=== PAUSE TestAccAWSEcsTaskDefinition_constraint
=== RUN   TestAccAWSEcsTaskDefinition_changeVolumesForcesNewResource
=== PAUSE TestAccAWSEcsTaskDefinition_changeVolumesForcesNewResource
=== RUN   TestAccAWSEcsTaskDefinition_arrays
=== PAUSE TestAccAWSEcsTaskDefinition_arrays
=== RUN   TestAccAWSEcsTaskDefinition_Fargate
=== PAUSE TestAccAWSEcsTaskDefinition_Fargate
=== RUN   TestAccAWSEcsTaskDefinition_ExecutionRole
=== PAUSE TestAccAWSEcsTaskDefinition_ExecutionRole
=== RUN   TestAccAWSEcsTaskDefinition_disappears
=== PAUSE TestAccAWSEcsTaskDefinition_disappears
=== RUN   TestAccAWSEcsTaskDefinition_Tags
=== PAUSE TestAccAWSEcsTaskDefinition_Tags
=== RUN   TestAccAWSEcsTaskDefinition_ProxyConfiguration
=== PAUSE TestAccAWSEcsTaskDefinition_ProxyConfiguration
=== RUN   TestAccAWSEcsTaskDefinition_inferenceAccelerator
=== PAUSE TestAccAWSEcsTaskDefinition_inferenceAccelerator
=== CONT  TestAccAWSEcsTaskDefinition_basic
=== CONT  TestAccAWSEcsTaskDefinition_withIPCMode
=== CONT  TestAccAWSEcsTaskDefinition_ExecutionRole
=== CONT  TestAccAWSEcsTaskDefinition_withEFSAccessPoint
=== CONT  TestAccAWSEcsTaskDefinition_withTaskScopedDockerVolume
=== CONT  TestAccAWSEcsTaskDefinition_withFsxWinFileSystem
=== CONT  TestAccAWSEcsTaskDefinition_Fargate
=== CONT  TestAccAWSEcsTaskDefinition_changeVolumesForcesNewResource
=== CONT  TestAccAWSEcsTaskDefinition_constraint
=== CONT  TestAccAWSEcsTaskDefinition_arrays
=== CONT  TestAccAWSEcsTaskDefinition_withPidMode
=== CONT  TestAccAWSEcsTaskDefinition_withEFSVolumeMinimal
=== CONT  TestAccAWSEcsTaskDefinition_ProxyConfiguration
=== CONT  TestAccAWSEcsTaskDefinition_withTransitEncryptionEFSVolume
=== CONT  TestAccAWSEcsTaskDefinition_withEcsService
=== CONT  TestAccAWSEcsTaskDefinition_withNetworkMode
=== CONT  TestAccAWSEcsTaskDefinition_withEFSVolume
=== CONT  TestAccAWSEcsTaskDefinition_inferenceAccelerator
=== CONT  TestAccAWSEcsTaskDefinition_withTaskRoleArn
=== CONT  TestAccAWSEcsTaskDefinition_Tags
--- PASS: TestAccAWSEcsTaskDefinition_withTaskScopedDockerVolume (52.87s)
=== CONT  TestAccAWSEcsTaskDefinition_withScratchVolume
--- PASS: TestAccAWSEcsTaskDefinition_arrays (53.27s)
=== CONT  TestAccAWSEcsTaskDefinition_withDockerVolumeMinimalConfig
--- PASS: TestAccAWSEcsTaskDefinition_inferenceAccelerator (55.70s)
=== CONT  TestAccAWSEcsTaskDefinition_disappears
--- PASS: TestAccAWSEcsTaskDefinition_withTaskRoleArn (59.59s)
=== CONT  TestAccAWSEcsTaskDefinition_withDockerVolume
--- PASS: TestAccAWSEcsTaskDefinition_withIPCMode (60.86s)
--- PASS: TestAccAWSEcsTaskDefinition_constraint (61.12s)
--- PASS: TestAccAWSEcsTaskDefinition_withPidMode (64.15s)
--- PASS: TestAccAWSEcsTaskDefinition_ExecutionRole (65.06s)
--- PASS: TestAccAWSEcsTaskDefinition_withNetworkMode (65.73s)
--- PASS: TestAccAWSEcsTaskDefinition_ProxyConfiguration (69.45s)
--- PASS: TestAccAWSEcsTaskDefinition_withTransitEncryptionEFSVolume (70.58s)
--- PASS: TestAccAWSEcsTaskDefinition_withEFSVolumeMinimal (73.25s)
--- PASS: TestAccAWSEcsTaskDefinition_withEFSVolume (73.75s)
--- PASS: TestAccAWSEcsTaskDefinition_withEFSAccessPoint (75.37s)
--- PASS: TestAccAWSEcsTaskDefinition_Fargate (76.75s)
--- PASS: TestAccAWSEcsTaskDefinition_changeVolumesForcesNewResource (82.88s)
--- PASS: TestAccAWSEcsTaskDefinition_basic (83.44s)
--- PASS: TestAccAWSEcsTaskDefinition_withDockerVolumeMinimalConfig (36.94s)
--- PASS: TestAccAWSEcsTaskDefinition_withScratchVolume (37.40s)
--- PASS: TestAccAWSEcsTaskDefinition_withDockerVolume (31.00s)
--- PASS: TestAccAWSEcsTaskDefinition_disappears (42.62s)
--- PASS: TestAccAWSEcsTaskDefinition_Tags (100.23s)
--- PASS: TestAccAWSEcsTaskDefinition_withEcsService (157.31s)
--- PASS: TestAccAWSEcsTaskDefinition_withFsxWinFileSystem (1420.86s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	1424.417s
```

##### GovCloud

```console
% make testacc TEST=./aws TESTARGS='-run=TestAccAWSEcsTaskDefinition_' 
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -count 1 -parallel 20 -run=TestAccAWSEcsTaskDefinition_ -timeout 180m
=== RUN   TestAccAWSEcsTaskDefinition_basic
=== PAUSE TestAccAWSEcsTaskDefinition_basic
=== RUN   TestAccAWSEcsTaskDefinition_withScratchVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withScratchVolume
=== RUN   TestAccAWSEcsTaskDefinition_withDockerVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withDockerVolume
=== RUN   TestAccAWSEcsTaskDefinition_withDockerVolumeMinimalConfig
=== PAUSE TestAccAWSEcsTaskDefinition_withDockerVolumeMinimalConfig
=== RUN   TestAccAWSEcsTaskDefinition_withEFSVolumeMinimal
=== PAUSE TestAccAWSEcsTaskDefinition_withEFSVolumeMinimal
=== RUN   TestAccAWSEcsTaskDefinition_withEFSVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withEFSVolume
=== RUN   TestAccAWSEcsTaskDefinition_withTransitEncryptionEFSVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withTransitEncryptionEFSVolume
=== RUN   TestAccAWSEcsTaskDefinition_withEFSAccessPoint
=== PAUSE TestAccAWSEcsTaskDefinition_withEFSAccessPoint
=== RUN   TestAccAWSEcsTaskDefinition_withFsxWinFileSystem
=== PAUSE TestAccAWSEcsTaskDefinition_withFsxWinFileSystem
=== RUN   TestAccAWSEcsTaskDefinition_withTaskScopedDockerVolume
=== PAUSE TestAccAWSEcsTaskDefinition_withTaskScopedDockerVolume
=== RUN   TestAccAWSEcsTaskDefinition_withEcsService
=== PAUSE TestAccAWSEcsTaskDefinition_withEcsService
=== RUN   TestAccAWSEcsTaskDefinition_withTaskRoleArn
=== PAUSE TestAccAWSEcsTaskDefinition_withTaskRoleArn
=== RUN   TestAccAWSEcsTaskDefinition_withNetworkMode
=== PAUSE TestAccAWSEcsTaskDefinition_withNetworkMode
=== RUN   TestAccAWSEcsTaskDefinition_withIPCMode
=== PAUSE TestAccAWSEcsTaskDefinition_withIPCMode
=== RUN   TestAccAWSEcsTaskDefinition_withPidMode
=== PAUSE TestAccAWSEcsTaskDefinition_withPidMode
=== RUN   TestAccAWSEcsTaskDefinition_constraint
=== PAUSE TestAccAWSEcsTaskDefinition_constraint
=== RUN   TestAccAWSEcsTaskDefinition_changeVolumesForcesNewResource
=== PAUSE TestAccAWSEcsTaskDefinition_changeVolumesForcesNewResource
=== RUN   TestAccAWSEcsTaskDefinition_arrays
=== PAUSE TestAccAWSEcsTaskDefinition_arrays
=== RUN   TestAccAWSEcsTaskDefinition_Fargate
=== PAUSE TestAccAWSEcsTaskDefinition_Fargate
=== RUN   TestAccAWSEcsTaskDefinition_ExecutionRole
=== PAUSE TestAccAWSEcsTaskDefinition_ExecutionRole
=== RUN   TestAccAWSEcsTaskDefinition_disappears
=== PAUSE TestAccAWSEcsTaskDefinition_disappears
=== RUN   TestAccAWSEcsTaskDefinition_Tags
=== PAUSE TestAccAWSEcsTaskDefinition_Tags
=== RUN   TestAccAWSEcsTaskDefinition_ProxyConfiguration
=== PAUSE TestAccAWSEcsTaskDefinition_ProxyConfiguration
=== RUN   TestAccAWSEcsTaskDefinition_inferenceAccelerator
=== PAUSE TestAccAWSEcsTaskDefinition_inferenceAccelerator
=== CONT  TestAccAWSEcsTaskDefinition_basic
=== CONT  TestAccAWSEcsTaskDefinition_ExecutionRole
=== CONT  TestAccAWSEcsTaskDefinition_inferenceAccelerator
=== CONT  TestAccAWSEcsTaskDefinition_ProxyConfiguration
=== CONT  TestAccAWSEcsTaskDefinition_Tags
=== CONT  TestAccAWSEcsTaskDefinition_disappears
=== CONT  TestAccAWSEcsTaskDefinition_withPidMode
=== CONT  TestAccAWSEcsTaskDefinition_constraint
=== CONT  TestAccAWSEcsTaskDefinition_Fargate
=== CONT  TestAccAWSEcsTaskDefinition_withIPCMode
=== CONT  TestAccAWSEcsTaskDefinition_withEFSAccessPoint
=== CONT  TestAccAWSEcsTaskDefinition_withNetworkMode
=== CONT  TestAccAWSEcsTaskDefinition_withTaskRoleArn
=== CONT  TestAccAWSEcsTaskDefinition_withEcsService
=== CONT  TestAccAWSEcsTaskDefinition_withTaskScopedDockerVolume
=== CONT  TestAccAWSEcsTaskDefinition_arrays
=== CONT  TestAccAWSEcsTaskDefinition_withEFSVolumeMinimal
=== CONT  TestAccAWSEcsTaskDefinition_withTransitEncryptionEFSVolume
=== CONT  TestAccAWSEcsTaskDefinition_withFsxWinFileSystem
=== CONT  TestAccAWSEcsTaskDefinition_changeVolumesForcesNewResource
=== CONT  TestAccAWSEcsTaskDefinition_inferenceAccelerator
    provider_test.go:1103: skipping test for aws-us-gov/us-gov-west-1: Error running apply: exit status 1
        2021/06/07 06:21:14 [DEBUG] Using modified User-Agent: Terraform/0.12.31 HashiCorp-terraform-exec/0.13.3
        
        Error: ClientException: Unsupported field 'inferenceAccelerators'.
        
          on terraform_plugin_test.tf line 2, in resource "aws_ecs_task_definition" "test":
           2: resource "aws_ecs_task_definition" "test" {
        
        
--- SKIP: TestAccAWSEcsTaskDefinition_inferenceAccelerator (6.15s)
=== CONT  TestAccAWSEcsTaskDefinition_withEFSVolume
--- PASS: TestAccAWSEcsTaskDefinition_arrays (65.16s)
=== CONT  TestAccAWSEcsTaskDefinition_withDockerVolume
--- PASS: TestAccAWSEcsTaskDefinition_withTaskScopedDockerVolume (66.20s)
=== CONT  TestAccAWSEcsTaskDefinition_withDockerVolumeMinimalConfig
--- PASS: TestAccAWSEcsTaskDefinition_constraint (68.34s)
=== CONT  TestAccAWSEcsTaskDefinition_withScratchVolume
--- PASS: TestAccAWSEcsTaskDefinition_withTaskRoleArn (74.48s)
--- PASS: TestAccAWSEcsTaskDefinition_withIPCMode (76.45s)
--- PASS: TestAccAWSEcsTaskDefinition_withNetworkMode (77.07s)
--- PASS: TestAccAWSEcsTaskDefinition_withPidMode (77.25s)
--- PASS: TestAccAWSEcsTaskDefinition_ExecutionRole (77.42s)
--- PASS: TestAccAWSEcsTaskDefinition_ProxyConfiguration (78.51s)
--- PASS: TestAccAWSEcsTaskDefinition_withEFSVolumeMinimal (78.64s)
--- PASS: TestAccAWSEcsTaskDefinition_withTransitEncryptionEFSVolume (81.40s)
--- PASS: TestAccAWSEcsTaskDefinition_Fargate (82.39s)
--- PASS: TestAccAWSEcsTaskDefinition_withEFSAccessPoint (83.54s)
--- PASS: TestAccAWSEcsTaskDefinition_disappears (85.26s)
--- PASS: TestAccAWSEcsTaskDefinition_withEFSVolume (59.82s)
--- PASS: TestAccAWSEcsTaskDefinition_basic (91.24s)
--- PASS: TestAccAWSEcsTaskDefinition_changeVolumesForcesNewResource (91.29s)
--- PASS: TestAccAWSEcsTaskDefinition_withDockerVolume (29.29s)
--- PASS: TestAccAWSEcsTaskDefinition_withScratchVolume (27.24s)
--- PASS: TestAccAWSEcsTaskDefinition_withDockerVolumeMinimalConfig (29.51s)
--- PASS: TestAccAWSEcsTaskDefinition_Tags (106.55s)
--- PASS: TestAccAWSEcsTaskDefinition_withEcsService (131.01s)
=== RUN   TestAccAWSEcsTaskDefinition_withFsxWinFileSystem
    resource_aws_ecs_task_definition_test.go:369: Amazon FSx for Windows File Server volumes for ECS tasks are not supported in GovCloud partition
--- SKIP: TestAccAWSEcsTaskDefinition_withFsxWinFileSystem (0.00s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	1127.866s
```

**Comment by ewbankkit** [comment]
@DrFaust92 Thanks for the contribution :tada: :clap:.
I pushed a minor change to suppress errors from GovCloud testing.

**Pull request merged**